### DISCUSSION TITLE
Problem with pre-release channels for existing tags

### DISCUSSION THREAD

**Discussion by ties-s** [semantic-release/semantic-release#2238] [Release workflow] [answered]
The docs mention using properly formatted tags should be sufficient for adding semantic release to existing repos. For pre-release channels however, I have found that semantic release relies on `git notes` attached to the tags to find existing versions.

This should either be changed or properly documented

**Comment by travi** [comment, marked as answer]
please see https://semantic-release.gitbook.io/semantic-release/support/troubleshooting#release-not-found-release-branch-after-git-push-force for details on how to resolve this situation.

if you have suggestions on how the documentation could be improved, we would welcome a contribution.

**Reply by ties-s** [reply]
Thank you! That fixed it for me.

Would be great if the notes part could also be mentioned [here](https://semantic-release.gitbook.io/semantic-release/usage/configuration#existing-version-tags) since that states dimply adding a version tag should make it work. I have added a little note in #2240.

**Comment by dominykas** [comment]
I do have a wip PR to address this, but not sure what are the chances of it getting merged if I were to finish it (tests are missing)... https://github.com/semantic-release/semantic-release/pull/1820
